cognite/client/_api/transformation_jobs.py

```diff
@@ -0,0 +1,103 @@
+from typing import List, Optional
+
+from cognite.client import utils
+from cognite.client._api_client import APIClient
+from cognite.client.data_classes import TransformationJob, TransformationJobList, TransformationJobMetricList
+
+
+class TransformationJobsAPI(APIClient):
+    _RESOURCE_PATH = "/transformations/jobs"
+    _LIST_CLASS = TransformationJobList
+
+    def list(self, limit: Optional[int] = 25, transformation_id: Optional[int] = None) -> TransformationJobList:
+        """`List all running transformation jobs. <https://docs.cognite.com/api/playground/#operation/transformationJobs>`_
+
+        Args:
+            limit (int): Limits the number of results to be returned. To retrieve all results use limit=-1, default limit is 25.
+            transformation_id (int): Filters the results by the internal transformation id.
+
+        Returns:
+            TransformationJobList: List of transformation jobs
+
+        Example:
+
+            List transformation jobs::
+
+                >>> from cognite.experimental import CogniteClient
+                >>> c = CogniteClient()
+                >>> transformation_jobs_list = c.transformations.jobs.list()
+
+            List transformation jobs of a single transformation::
+
+                >>> from cognite.experimental import CogniteClient
+                >>> c = CogniteClient()
+                >>> transformation_jobs_list = c.transformations.jobs.list(transformation_id = 1)
+        """
+        if transformation_id is not None:
+            resource_path = utils._auxiliary.interpolate_and_url_encode(
+                "/transformations/{}/jobs", str(transformation_id)
+            )
+            return self._list(method="GET", limit=limit, resource_path=resource_path)
+
+        return self._list(method="GET", limit=limit)
+
+    def retrieve(self, id: int) -> Optional[TransformationJob]:
+        """`Retrieve a single transformation job by id. <https://docs.cognite.com/api/playground/#operation/getTransformationJob>`_
```

Comment:
this should be pointing to this https://github.com/cognitedata/service-contracts/pull/1501/files#diff-8df5cf6e84a989d49d837ea04347ac6085957197f0ef01467745af40090de15dR1322

https://docs.cognite.com/api/v1/#operation/getApiV1ProjectsProjectTransformationsJobsJobid

again, see if we can shorten the op id.